boards/avr-rss2/doc.txt

```diff
@@ -0,0 +1,103 @@
+/**
+@defgroup    boards_avr-rss2 Atmega256RFR2 rss2
+@ingroup     boards
+@brief       Support for the Radio Sensors Atmega256RFR2 rss2 board
+
+### General information
+Document version V1.4 2019-11-10<br>
+Board is based AtMega256RFR2 with additional HW, EUI64 chip, HW comparator, RTC XTAL, BME280
+
+- Directory `$(RIOTBASE)/boards/avr-rss2'
+
+### Board Features
+
+- PCB Antenna. Supercardiod
+- Robust radio performance. Up to 300 meter line-of-sight
+- Unique EUI64 address via 24MAC602 chip with 2k EEPROM
+- Rev 2.4 Onboard BME280 for Temp/RH/Pressure
+- Light Sensor
+- 32kHz RTC clock xtal
+- HW comparator chip and input
+- Programmable power FET, (relay) for power on/off of sensors
+- Up to 25V DC input via LDO (LP2950). TVS protected
+- Standard. 6-pin TTL-USB header compatible w. FTDI cable for USART
+- PCB formfactor for cheap project box G.40X IP54
+- Power/current use:
+*- RX ~10mA (Full RPC AtMegaXXRFR2)
+*- Sleep ~45uA @ 16MHz XTAL
+*- Sleep ~15uA @  8MHz using internal oscillator
+- Preprogammed Atmel standard stk500v2 bootloader
+- CE certified by test institute
+
+### USART
+The board has one USART via the 6-pin TTL-USB adapter, The default
+baudrate is 115200 bps. It's possible to use higher speeds as is 250k
+and 500k baud which gives 0% Error with 16MHz clock.
+(An additional USART is on the chip but as default used by interrupt
+pins. Needs HW mod to be used)
+
+### Toolchain
+The Atmel toolchain is available in most operating systems.
+For a full toolchain and easy installation on Ubuntu:
+```
+apt-get install gcc-avr avr-libc avrdude
+```
+Otherwise if OS toolchain packages do not support the AtMega256RfR2
+MCU, another option is to download the C compiler toolchain from Microchip.
+
+### For Linux and MacOS
+1. Download the proper 8-bit C compiler AVR toolchain, 32 or 64-bit, [from Microchip](https://www.microchip.com/mplab/avr-support/avr-and-arm-toolchains-c-compilers).
+2. Unpack under `/usr/local`
+3. Add to your search PATH. For example add to `.bashrc`:
+```
+export PATH=$PATH:/usr/local/avr8-gnu-toolchain-linux_x86_64/bin (64-bit)
+export PATH=$PATH:/usr/local/avr8-gnu-toolchain-linux_x86/bin (32-bit)
+```
+4. For flash programming, you need `avrdude`. On Ubuntu Linux, it can be installed with the command
+
+```
+apt-get install avrdude
+```
+For MacOS, there is a [HomeBrewpackage](https://formula.brew.sh/formula/avrdude#default)
+that can be installed with the command:
+```
+brew install avrdude
+```
+
+### RIOT build
+
+```
+make BOARD=avr-rss2
+```
+
+### Flashing/Programming hardware
+Programming using avrdude using serial bootloader using a TTL-USB cable.
+Press the RESET button. The bootloader with wait for boot commands for 3 seconds. Program with:
```

Comment:
> Press the RESET button.

You should highlight this more.
With most boards that are flashed over UART the reset is triggered automatically by e.g. the CTS line.

At first I thought I connected it wrong or broke it by using 5V Vcc, but then I read the docs more carefully again.

Probably doesn't hurt to mention that the Vcc pin on the FTDI header is connected to the LDO, so supplying 5V is perfectly fine.